code/game/objects/items.dm

```diff
@@ -53,6 +52,10 @@
 	var/vending_cat = null// subcategory for vending machines.
 	var/list/dynamic_overlay[0] //For items which need to slightly alter their on-mob appearance while being worn.
 
+	var/is_shrapnel = 0
+	var/shrapnel_amount = 0
+	var/shrapnel_type = ""
```

Comment:
So instead of `shrapnel_type = "/obj/item/projectile/bullet/shrapnel"` you'd have just `shrapnel_type = /obj/item/projectile/bullet/shrapnel`